### DISCUSSION TITLE
Include 'RunAfterStaging' option in RDMP

### DISCUSSION THREAD

**Discussion by sfstalin** [HicServices/RDMP#558] [Ideas]
**Is your feature request related to a problem? Please describe.**
When a data load fails after loading RAW (a long time e.g. 100 million records)

**Describe the solution you'd like**
The ability to resume the data load from the 'RAW is already populated' stage

**Comment by tznind** [comment]
I've converted this to a discussion because it would take a lot of work to make this happen and that would mean dedicated resource being allocated to this feature.
